static/css/blog_post_content_next.css

```
@@ -0,0 +1,206 @@
+*/
+
+.blog-post-content {
+  max-width: 878px;
+  margin: 0 auto;
+  color: #303030;
+  font-size: 22px;
+  font-weight: 300;
+  font-family: "Noto Sans KR", AppleGothic, Dotum, sans-serif;
+  line-height: 1.75;
+  letter-spacing: -0.66px;
+  word-break: break-word;
```

Stylelint (17.4.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)